src/server/scripts/Outland/zone_shattrath_city.cpp

```diff
@@ -471,8 +518,8 @@ class npc_ishanah : public CreatureScript
         if (creature->IsQuestGiver())
             player->PrepareQuestMenu(creature->GetGUID());
 
-        player->ADD_GOSSIP_ITEM(GOSSIP_ICON_CHAT, ISANAH_GOSSIP_1, GOSSIP_SENDER_MAIN, GOSSIP_ACTION_INFO_DEF+1);
-        player->ADD_GOSSIP_ITEM(GOSSIP_ICON_CHAT, ISANAH_GOSSIP_2, GOSSIP_SENDER_MAIN, GOSSIP_ACTION_INFO_DEF+2);
+        player->ADD_GOSSIP_ITEM_DB(MENU_OPTION_WHO_ARE_THE_SHATAR, OPTION_ID_WHO_ARE_THE_SHATAR, GOSSIP_SENDER_MAIN, GOSSIP_ACTION_INFO_DEF+1);
+        player->ADD_GOSSIP_ITEM_DB(MENU_OPTION_WHY_DO_YOU_ALLOW,   OPTION_ID_WHY_DO_YOU_ALLOW,   GOSSIP_SENDER_MAIN, GOSSIP_ACTION_INFO_DEF+2);
```

Comment:
this is doable with gossip_* tables only, no conditions or sai, no need for cpp script